server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotRequest.java

```diff
@@ -63,6 +66,7 @@
  */
 public class CreateSnapshotRequest extends MasterNodeRequest<CreateSnapshotRequest>
         implements IndicesRequest.Replaceable, ToXContentObject {
+    public static int MAXIMUM_METADATA_BYTES = 1024; // chosen arbitrarily
```

Comment:
Do we test it somewhere?

Reply:
Yes, in `CreateSnapshotRequestTests`:

https://github.com/elastic/elasticsearch/blob/eeadd6547d0fdc1dbfc699922af406296b52408b/server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/create/CreateSnapshotRequestTests.java#L122